streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -1209,4 +1211,46 @@ public void shouldNotMeasureLatencyBecauseSensorHasNoMetrics() {
 
         verify(sensor);
     }
+
+    @Test
+    public void shouldAddThreadLevelMutableMetric() {
+        final StreamsMetricsImpl streamsMetrics
+            = new StreamsMetricsImpl(metrics, THREAD_ID1, VERSION, time);
+
+        streamsMetrics.addThreadLevelMutableMetric(
+            "foobar",
+            "test metric",
+            "t1",
+            (c, t) -> 123
```

Comment:
Could you please use variables with a meaningful name instead of a literal and re-use the variable in the verification?